src/Storage/Imap.php

```diff
@@ -354,16 +354,31 @@ public function getFolders($rootFolder = null)
      * @throws Exception\RuntimeException
      * @throws \Zend\Mail\Protocol\Exception\RuntimeException
      */
-    public function selectFolder($globalName)
+    public function selectFolder($globalName, $examine = false)
```

Comment:
Would you rather have `examineFolder()` be identical to `selectFolder()` aside from the examine/select?  i.e. copy/pasting `selectFolder()` and using examine instead?